src/main/software/amazon/event/ruler/Task.java

```diff
@@ -46,39 +47,39 @@ class Task {
     }
 
     NameState startState() {
-        return machine.getStartState();
+        return this.machine.getStartState();
```

Comment:
Why? (recurs)

Reply:
using ```this.field``` gives more understanding that this field is related to the class. The same with methods.

Reply:
I'd recommend to not change this here as nearly all other classes aren't using `this.`

I want the codebase to stay consistent first, and not be deviate unless there's a code reason for one class to be unique on its own.